homepage/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from django.forms import ModelForm
+from .models import Review
+
+
+class ReviewForm(ModelForm):
+    class Meta:
+        model = Review
+        fields = ['course', 'user', 'rate', 'content', 'course_load', 'professor']
+        CHOICES = [['', '---'], [1, '1'], [2, '2'], [3, '3'], [4, '4'], [5, '5']]

```

Comment:
Why ['','---'] is one of the options? Both fields are mandatory
Is the first option the one that is displayed as default?
I'm not sure how the choices dropdown works yet but I think we should avoid allowing the user to leave them empty,
even if that means that by default the rating will be 3 or something

Reply:
Where are the other 'preset' fields for the dorm, such as getting a course from the approved list and such? are we leaving them for after the demo?

Reply:
![image](https://user-images.githubusercontent.com/57869309/116524113-ad15ea00-a8df-11eb-9db2-e81ed456d989.png)
@ellaml The ```''``` makes the user to chose an option, the ```'---'``` is what rendered, I tried to be consistent with the other fields. So the user must choose a number to post a review.

Reply:
![image](https://user-images.githubusercontent.com/57869309/116524873-8f955000-a8e0-11eb-8389-e54a62980f64.png)
@Vera Django gives us a gift since we made the connection between our models, by default the form shows options only from the approved list.

Reply:
great!

Reply:
awesome :)